src/libYARP_OS/src/Node.cpp

```diff
@@ -350,6 +436,10 @@ void yarp::os::Node::Helper::add(Contactable& contactable)
     }
     prepare(name);
     item.contactable = &contactable;
+    ROSReport* rep = new ROSReport();
+    item.contactable->setReporter(*rep);
```

Comment:
@Tobias-Fischer I'm just worried that someone could call `setReporter()` and replace the `ROSReport` with a custom one. If this happens, then the ROS Node will no longer work... I don't know if the `Contactable& contactable` is accessible in some way from outside, though. Do you know if this is possible?